pkg/capabilities/consensus/ocr3/ocr3_common-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/common",
+  "$defs": {
+    "SignedReport": {
+      "type": "object",
+      "properties": {
+        "WorkflowExecutionID": {
+          "type": "string"
+        },
+        "Value": {
+          "type": "object",
+          "properties": {
+            "Underlying": {
+              "type": "object"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["Underlying"]
+        },
+        "Err": {
+          "type": "boolean"
```

Comment:
@nolag Is this correct?

Reply:
No, I had another PR that fixed it.  I merged my own conflict in the latest commit :)